### ISSUE TITLE
GRPC: Implement near  feature parity with cli

### ISSUE THREAD

**Issue by TheCharlatan** [farcaster-project/farcaster-node#471]
The grpc api should be similarly powerful as the cli. Most commands that are available on the cli have to ported for grpc.

**Comment by h4sh3d** [comment]
Currently building the GUI and experimenting the grpcd service, we currently don't have the possibility to get the listening peers `swap-cli ll`. This could go in another PR after #739

**Comment by TheCharlatan** [comment]
> Currently building the GUI and experimenting the grpcd service, we currently don't have the possibility to get the listening peers `swap-cli ll`. This could go in another PR after #739

I'm not sure if we should port this one over. The listens are available in the normal `info` request.

**Comment by h4sh3d** [comment]
> I'm not sure if we should port this one over. The listens are available in the normal `info` request.

Based on my tests `info` gives you only the connected peers, not your listeners. After making an offer the info peers list stays empty but `ll` gives you the list with `0.0.0.0:9735`

**Comment by TheCharlatan** [comment]
Mmh, seems to work fine for me. Info is called right after make offer:

```
info response: InfoResponse { id: 0, listens: ["0.0.0.0:9376"], uptime: 60, since: 1668069380, peers: [], swaps: [], offers: ["Offer:Cke4ftrP5A7XVkpjXrJwrRWM6ah2PciF11W723UjzEWsNR4gdPRjgF911111uMFfQgccTXS3uFsEj11111TBMkqM113GTyRhcZM1111114A4TT6VZ9wi2eQDnd9diZ1KpTH3CZGbf7FHijo4K35QfqisGy11111111111111111111111111111111111111111AfZ113WigirQSg1N"] }
ok
```

**Comment by h4sh3d** [comment]
It's my bad, I was using `.getPeers()` and missed the `.getListens()`
